packages/common/driver-definitions/src/storage.ts

```diff
@@ -91,6 +91,9 @@ export interface IDocumentDeltaStorageService {
     ): IStream<ISequencedDocumentMessage[]>;
 }
 
+// DO NOT INCREASE THIS TYPE'S VALUE - If a driver started using a larger value, GC would likely start closing sessions
+export type FiveDaysMs = 432000000; /* 5 days in milliseconds */
```

Comment:
nit: Should this be called `FiveDaysMsType`? When I was reading the code that uses it, I got a little confused and thought this was a value rather than a type.

Reply:
I prefer not to put "Type" in type names. But I see what you mean!